indico/modules/events/registration/templates/management/registration_tag_assign.html

```diff
@@ -0,0 +1,14 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+{% from 'message_box.html' import message_box %}
+
+<div class="registration-tags">
+    {{ form_header(form, id='registration-tag-assign-form') }}
```

Comment:
This ID doesn't seem to be used anywhere

```suggestion
    {{ form_header(form) }}
```

Reply:
I'd delete this template as well but it's by default rendered too wide so I need to somehow keep the `.registration-tags` class to limit the width..

Reply:
IF setting it on the form itself works, you could try passing it via `form_header_kwargs` to `jsonify_form`